src/easylogging++.h

```diff
@@ -6751,6 +6751,7 @@ if (ELPP_DEBUG_LOG) C##LEVEL##_EVERY_N(el::base::Writer, n, el::base::DispatchAc
 //
 // Default Debug Only Loggers macro using CLOG(), CLOG_VERBOSE() and CVLOG() macros
 //
+#if !defined(ELPP_DISABLE_DEBUG_MACROS)
```

Comment:
word `disable` suggest it's soft (meaning it's not going to break the code) you should rename it to `ELPP_NO_DEBUG_MACROS`.

Let me explain what i mean by soft. When code is using `DLOG` defining `ELPP_DISABLE_DEBUG_MACROS` should compile code just fine just not have any affect at the `DLOG` lines. Where as user would know `ELPP_NO_DEBUG_MACROS` means it's going to break the code. 

Open for discussion though if anyone disagree